docs/tutorials-and-quickstart-guides/openms-user-tutorial.md

```diff
@@ -18,48 +18,44 @@ OpenMS User Tutorial
 ### Installation
 
 Before we get started, we will install OpenMS and KNIME. If you take part in a training session you will have likely received an USB stick from us that contains the required data and software. If we provide laptops with the software you may of course skip the installation process and continue reading the next section.
+If you are doing this tutorial online, choose online in the following tab(s).
 
-#### Installation from the OpenMS USB stick
+::::{tab-set}
+
+:::{tab-item} Online
+:sync: online
+
+If you are working through this tutorial at home/online, proceed with the following steps:
+
+- Download and install OpenMS using the installation instructions for your operating system:
+  - [GNU/Linux](/installations/installation-on-gnu-linux.md)
+  - [macOS](/installations/installation-on-macos.md)
+  - [Windows](/installations/installation-on-windows.md)
+- Download and install [KNIME](https://www.knime.org/downloads/overview)
+
+:::
+
+:::{tab-item} USB Stick
+:sync: usb
```

Comment:
I have seen the final outcome  [](https://openms.readthedocs.io/en/latest/tutorials-and-quickstart-guides/openms-user-tutorial.html ) and checked the tabs and links, they are working fine, especially the use of::::{tab-set} and :::{tab-item} within the document that enables user selection between installing from USB or online.